components/TripleCounterButton/index.js

```diff
@@ -0,0 +1,76 @@
+import { StyledButton } from "../../styles/StyledButton";
+import { StyledSpan } from "../../styles/StyledSpan";
+import { StyledInstruction2 } from "../../styles/StyledInstruction2";
+import useStore from "../useStore";
+import useLoading from "../useLoading";
+
+export default function TripleCounterButton({
+  stopTripleCounter,
+  stopTripleCounterMusic,
+  playGong,
+  playTripleCounter,
+  playTripleCounterMusic,
+  router,
+}) {
+  const tripleCounterOne = useStore((state) => state.tripleCounterOne);
+  const tripleCounterTwo = useStore((state) => state.tripleCounterTwo);
+  const tripleCounterThree = useStore((state) => state.tripleCounterThree);
+  const isTripleCounterOneActive = useStore(
+    (state) => state.isTripleCounterOneActive
+  );
+  const isTripleCounterTwoActive = useStore(
+    (state) => state.isTripleCounterTwoActive
+  );
+  const isTripleCounterThreeActive = useStore(
+    (state) => state.isTripleCounterThreeActive
+  );
+  const switchIsTripleCounterOneActive = useStore(
+    (state) => state.switchIsTripleCounterOneActive
+  );
+  const isLoading = useStore((state) => state.isLoading);
+  const switchIsLoading = useStore((state) => state.switchIsLoading);
+
+  useLoading(switchIsLoading);
+
+  return (
+    <>
+      <StyledButton
+        onClick={() => {
+          if (!isLoading) {
+            isTripleCounterOneActive ||
+            isTripleCounterTwoActive ||
+            isTripleCounterThreeActive
+              ? (router.push("/success"),
+                stopTripleCounter(),
+                stopTripleCounterMusic(),
+                playGong())
+              : (switchIsTripleCounterOneActive(true),
+                playTripleCounterMusic(),
+                playTripleCounter());
+          }
```

Comment:
I would suggest to outsource this into a separate `handle` function and pass only this handle function here.